sql_generators/mobile_kpi_support_metrics/templates/new_profile_clients.view.sql

```diff
@@ -0,0 +1,34 @@
+{{ header }}
```

Comment:
Sorry for my ignorance about whats possible here, but would it practical to add a test that checks whether the grain of the table (or partition) is one row per client_id?

Reply:
This is a view and therefore we do not have the ability to do this yet (limitation of bqetl checks design).